app/config/init.js

```diff
@@ -0,0 +1,43 @@
+const vm = require('vm');
+const notify = require('../notify');
+
+const _extract = function (script) {
+  const module = {};
+  script.runInNewContext({module});
+  if (!module.exports) {
+    throw new Error('Error reading configuration: `module.exports` not set');
+  }
+  return module.exports;
+};
+
+const _syntaxValidation = function (cfg) {
```

Comment:
haha the comment was to update _syntaxValidation to the code i wrote in my comment,
the blocks under is to show what it does